armsrc/Standalone/Makefile.inc

```diff
@@ -33,11 +33,11 @@ endif
 ifneq (,$(findstring WITH_STANDALONE_HF_BOG,$(APP_CFLAGS)))
     SRC_STANDALONE = hf_bog.c
 endif
-# WITH_STANDALONE_HF_14ASNIFF
-ifneq (,$(findstring WITH_STANDALONE_HF_14ASNIFF,$(APP_CFLAGS)))
-    SRC_STANDALONE = hf_14asniff.c
+# WITH_STANDALONE_LF_EM4100EMUL
+ifneq (,$(findstring WITH_STANDALONE_LF_EM4100EMUL,$(APP_CFLAGS)))
+    SRC_STANDALONE = lf_em4100emul.c
 endif
-# WITH_STANDALONE_LF_ICEHID
-ifneq (,$(findstring WITH_STANDALONE_LF_ICEHID,$(APP_CFLAGS)))
-    SRC_STANDALONE = lf_icehid.c
+# WITH_STANDALONE_LF_EM4100RWC
```

Comment:
You are replacing a standalonemode here.   I suggest you add your own after the last